pure.zsh

```diff
@@ -517,7 +517,9 @@ prompt_pure_state_setup() {
 		who_out=$(who -m 2>/dev/null)
 		if (( $? )); then
 			# Who am I not supported, fallback to plain who.
-			who_out=$(who 2>/dev/null | grep ${TTY#/dev/})
+			local -a who_in
+			who_in=( ${(f)"$(who 2>/dev/null)"} )
+			who_out="${(M)who_in:#*[:space:]${TTY#/dev/}[:space:]*}"
```

Comment:
Hmm, `[:space:]` doesn't seem to work in my shell, does it require some option to be enabled?

Reply:
Ah, seems to work with `[[:space:]]`.

Reply:
Ah, found it in the manual:

> Note that the square brackets are additional to those enclosing the whole set of characters,  so  to  test  for  a single alphanumeric character you need `[[:alnum:]]`.  Named character sets can be used alongside other types, e.g. `[[:alpha:]0-9]`.